design/src/main/java/com/urlaunched/android/design/ui/progressbarcontainer/ProgressBarContainer.kt

```diff
@@ -45,6 +48,59 @@ fun ProgressBarContainer(
     }
 }
 
+@Composable
+fun ProgressBarContainer(
+    modifier: Modifier = Modifier,
+    isLoading: Boolean,
+    indicatorColor: Color = ProgressIndicatorDefaults.circularColor,
+    indicatorTrackColor: Color = ProgressIndicatorDefaults.circularTrackColor,
+    progress: Float = 0F,
+    progressSettings: ProgressSettings,
+    content: @Composable () -> Unit
+) {
+    Box(modifier = modifier.fillMaxSize()) {
+        content()
+
+        if (isLoading) {
+            Box(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .background(Color.Black.copy(alpha = 0.75f))
```

Comment:
```suggestion
                    .background(Color.Black.copy(alpha = ProgressBarContainerConstants.CONTAINER_COLOR_ALPHA))
```

Reply:
Why do you need to use exactly 0.75f? The other ProgressBarContainer uses 0.65f as alpha?

Reply:
Because otherwise it's harder to read the text that is in white.